Loop/Managers/LoopAppManager.swift

```diff
@@ -230,6 +230,9 @@ class LoopAppManager: NSObject {
     // MARK: - Life Cycle
 
     func didBecomeActive() {
+        if let rootViewController = rootViewController {
+            ProfileExpirationAlerter.alertIfNeeded(viewControllerToPresentFrom: rootViewController)
```

Comment:
How does this interact with onboarding if onboarding is still displayed (yes, unlikely)?